common_spree_dependencies.rb

```diff
@@ -35,6 +35,7 @@
   gem 'with_model'
   gem 'rspec_junit_formatter'
   gem 'rails-controller-testing'
+  gem 'ffaker'
```

Comment:
Could we add `require: false` here? We don't need to require `ffaker` every time we boot the dummy apps.